package.json

```diff
@@ -23,9 +23,16 @@
     "@angular/platform-server": "^17.3.0",
     "@angular/router": "^17.3.0",
     "@angular/ssr": "^17.3.3",
+    "@stomp/stompjs": "^7.0.0",
+    "@types/socket.io": "^3.0.2",
+    "@types/stompjs": "^2.3.9",
+    "angular-sockjs": "^0.1.0",
     "bootstrap": "^5.3.3",
     "express": "^4.18.2",
     "rxjs": "~7.8.0",
+    "sockjs": "^0.3.24",
```

Comment:
This library is not necessary, it is used in the backend, the one we use in the frontend is sockjs-client which is already installed in the following line